src/main/java/github/scarsz/discordsrv/util/WebhookUtil.java

```diff
@@ -132,6 +132,7 @@ public static void deliverMessage(TextChannel channel, OfflinePlayer player, Str
                     .replace("%username%", player.getName())
                     .replace("%message%", message.replace("[", "\\["));
             chatMessage = PlaceholderUtil.replacePlaceholdersToDiscord(chatMessage, player);
+            chatMessage = DiscordUtil.translateEmotes(message, channel.getGuild());
```

Comment:
Should this not be `chatMessage = DiscordUtil.translateEmotes(chatMessage, channel.getGuild());`?